rfcs/README.md

```diff
@@ -0,0 +1,108 @@
+# Block Protocol RFCs
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for new features to enter the project.
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus approved by the Block Protocol Core Team.
+
+[Active RFCs](https://github.com/blockprotocol/blockprotocol/discussions/categories/rfc)
+[Proposed RFCs](https://github.com/blockprotocol/blockprotocol/pulls?q=is%3Aopen+is%3Apr+label%3A%22category%3A+rfc%22)
```

Comment:
This would cover not just proposed RFCs, but any pending/unmerged change to an RFC, right? (e.g. it may contain low-value submissions, uninteresting standalone typo fixes, etc... although hopefully those wouldn't remain "open" for long). Not sure this needs changing, just clarifying our expectations around this.

Reply:
It depends on how we use the label, we could create a few labels to distinguish between the various categories. But generally yes any open PR that is tagged as "rfc" will be shown in this view. I'm not sure that's too unfortunate as the main thing people want to look at are "Active RFCs", which aren't necessarily implemented yet (just have been accepted)